### PR TITLE
a3s authorization init

### PR DESCRIPTION
**Pull request by akorneyc**
This PR includes tilkynna integration to A3S instead of keycloak.

### CODE REVIEW

File: doc/quickstart_using_api/via_cli_curl.md [hunk 1]
```diff
@@ -15,16 +15,16 @@
...
+-s -v \
+-X POST http://localhost:80/connect/token \
 -H 'Content-Type: application/x-www-form-urlencoded' \
--H 'Bearer-Token: 575c92bb-33fe-45b8-85ef-7cc5710e62eb' \
 -H 'cache-control: no-cache' \
--d 'grant_type=password&username=test-user&password=test-user&client_id=tilkynna&client_secret=ee5c1c57-bf2f-43e6-9025-49344113c88d' \
+-d 'grant_type=password&username=tilkynna-admin&password=Password1#&client_id=tylkinna-test-client&client_secret=secret&scope=tilkynna' \
 | jq '.access_token' -r` \
-&& echo "TOKEN is: $TOKEN"
+&& echo "TOKEN is :$TOKEN"
```
Comment (melissapalmer) [inline doc/quickstart_using_api/via_cli_curl.md]: Is this tiny space chane correct? Does the token display ok, without an extra colon

File: src/main/java/org/tilkynna/security/SecurityContextUtility.java [hunk 2]
```diff
@@ -7,22 +7,25 @@
 package org.tilkynna.security;
```
Comment (melissapalmer) [inline src/main/java/org/tilkynna/security/SecurityContextUtility.java]: What are you not using the shared lib for this? and other such classes as AccessTokenPermissionsService

File: src/main/java/org/tilkynna/security/config/DisableSecurity.java [hunk 3]
```diff
@@ -15,10 +13,10 @@
 @Configuration
 @EnableWebSecurity
 @ConditionalOnProperty(prefix = "rest.security", value = "enabled", havingValue = "false")
-public class DisableSecurity extends WebSecurityConfigurerAdapter {
+public class DisableSecurity extends WebSecurityConfigurerAdapter{
     @Override
     protected void configure(HttpSecurity http) throws Exception {
         http.authorizeRequests().antMatchers("/").permitAll();
-        http.csrf().disable().authorizeRequests().anyRequest().permitAll();
+        http.csrf().disable() .authorizeRequests() .anyRequest().permitAll();
```
Comment (melissapalmer) [inline src/main/java/org/tilkynna/security/config/DisableSecurity.java]: Why is formatting changing, please check you using the formatting settings within project.

File: quickstart/docker-compose/docker-compose.dev.yml [hunk 4]
```diff
@@ -0,0 +1,108 @@
+#
```
Comment (melissapalmer) [inline quickstart/docker-compose/docker-compose.dev.yml]: Is this a duplicate of base docker-compose.yml?  Can we not do this as an override type thing

File: src/main/java/org/tilkynna/security/SecurityContextUtility.java (hunk 2 above)
Comment (akorneyc) [reply src/main/java/org/tilkynna/security/SecurityContextUtility.java]: hm... this approach was chosen as dokuti also on uses shared lib now. Right now do not remember why it not using, it was changed long ago, but I recall that repository no more exists in git...

File: doc/quickstart_using_api/via_cli_curl.md (hunk 1 above)
Comment (akorneyc) [reply doc/quickstart_using_api/via_cli_curl.md]: fixed

File: quickstart/docker-compose/docker-compose.dev.yml (hunk 4 above)
Comment (akorneyc) [reply quickstart/docker-compose/docker-compose.dev.yml]: fixed

File: src/main/java/org/tilkynna/security/config/DisableSecurity.java (hunk 3 above)
Comment (akorneyc) [reply src/main/java/org/tilkynna/security/config/DisableSecurity.java]: fixed

File: src/main/java/org/tilkynna/security/SecurityContextUtility.java (hunk 2 above)
Comment (akorneyc) [reply src/main/java/org/tilkynna/security/SecurityContextUtility.java]: right, repository no more exists but lib exists in artifactory and  this is not good using it

Comment (melissapalmer) [reply src/main/java/org/tilkynna/security/SecurityContextUtility.java]: These are due to be created within the A3S repo now. We need to check with @gm7 and @VictorSzewczenko as to what is happening with that at the moment.

**Pull request merged**